sqldelight-gradle-plugin/src/test/custom-dialect/dialect/src/main/kotlin/foo/FooDialect.kt

```diff
@@ -1,20 +1,31 @@
 package foo
 
-import app.cash.sqldelight.dialect.api.DialectType
-import app.cash.sqldelight.dialect.api.IntermediateType
-import app.cash.sqldelight.dialect.api.SqlDelightDialect
-import app.cash.sqldelight.dialect.api.TypeResolver
-import app.cash.sqldelight.dialect.api.encapsulatingType
-import app.cash.sqldelight.dialects.sqlite_3_18.SqliteDialect
-import app.cash.sqldelight.dialects.sqlite_3_18.SqliteTypeResolver
+import app.cash.sqldelight.dialect.api.*
+import app.cash.sqldelight.dialects.sqlite_3_18.*
 import com.alecstrong.sql.psi.core.psi.SqlFunctionExpr
 import com.alecstrong.sql.psi.core.psi.SqlTypeName
 import com.squareup.kotlinpoet.ClassName
 import com.squareup.kotlinpoet.CodeBlock
 import com.squareup.kotlinpoet.MemberName
 import com.squareup.kotlinpoet.TypeName
+import org.sqlite.*
+import java.io.*
+import java.sql.*
 
 class FooDialect : SqlDelightDialect by SqliteDialect() {
+  override val connectionManager: ConnectionManager = object: ConnectionManager by SqliteConnectionManager() {
```

Comment:
You would only need this 10 lines, the existing lines from `FooDialect` are unrelated to your use-case.